ichub-backend/main.py

```diff
@@ -151,7 +66,7 @@ def get_arguments():
      _/ // /__/_____/ __  / /_/ / /_/ /  / /_/ / /_/ / /__/ ,< /  __/ / / / /_/ /  
     /___/\____/    /_/ /_/\__,_/_.___/  /_____/\__,_/\___/_/|_|\___/_/ /_/\__,_/   
     """)
-    print("\n\n\t\t\t\t\t\t\t\t\t\tv0.0.1")
+    print("\n\n\t\t\t\t\t\t\t\t\t\tv0.1.3")
```

Comment:
The image tag is 0.0.1
```suggestion
    print("\n\n\t\t\t\t\t\t\t\t\t\tv0.0.1")
```